src/SHLD_editor.h

```diff
@@ -26,7 +26,7 @@ class SHLD_ctrl
 	//set's the control's value
 	void set_value(const std::vector<pcs_shield_triangle>&t){
 		data=t;
-		info->set_value(std::string(wxString::Format(_("Editing of shield data is currently unavailable\nshield has %d faces"), data.size()).mb_str()));
+		info->set_value(std::string(wxString::Format(_("Editing of shield data is currently unavailable\nshield has %ld faces"), data.size()).mb_str()));
```

Comment:
This will probably not work across platforms since a `size_t` (I'm assuming `data.size()` returns a `size_t`) does not have a constant size of 64-bits. In FSO we have a macro for handling that better (e.g. for [MSVC](https://github.com/scp-fs2open/fs2open.github.com/blob/master/code/globalincs/toolchain/msvc.h#L74) and [GCC](https://github.com/scp-fs2open/fs2open.github.com/blob/master/code/globalincs/toolchain/gcc.h#L60)). Maybe wxWidgets has something similar.